darky/bull-repl
# Fix: unresponsive commands when using --url 'rediss:...' to connect

# Motivation

Be able to use bull-repl with rediss protocol URLs.

# Description

While trying out the tool, noticed that it worked as expected for redis URLs, but did not result in commands producing any output when I connected to an URL with rediss protocol.

As it was not really possible to use the cli tool with rediss url's provided via **--url**, decided to add such possibility with a slight change done to the options preparation.

# Steps

Note that the behaviour can be seen also with these fake credentials (given below), but the no-output situation will disappear when steps are repeated with real credentials and this branch.

1. Use latest **master** branch bull-repl
2. connect to rediss url:<br>`bull-repl connect --url 'rediss://someUser:somePwd@some.host.com:12345' 'some.bull.queue'`
3. (bull-repl prompt appears and you're reported to have established a connection)
4. use command: `stats`

## Expected

1. output is generated with queue job stats

## Actual

1. nothing is displayed. the process just hangs without any output.
